package.json

```diff
@@ -6,9 +6,9 @@
     "node": ">=14.0.0"
   },
   "bin": {
-    "electron-markdownlint": "./dist/bin/markdownlint-cli-wrapper.js",
-    "electron-lint-markdown-links": "./dist/bin/lint-markdown-links.js",
-    "electron-lint-markdown-standard": "./dist/bin/lint-markdown-standard.js",
+    "lint-roller--markdownlint": "./dist/bin/markdownlint-cli-wrapper.js",
```

Comment:
If this is a good time to change it, I'm unreasonably bugged by the fact that there's no `-` between `markdown` and `lint` even though there is one in the other commands

Reply:
Thanks for this comment, it helped me catch this mistake - I'd actually meant to leave that named as `electron-markdownlint` but it got caught in a search and replace (hence the double `--` typo).

The `markdownlint` here is referring to [`DavidAnson/markdownlint`](https://github.com/DavidAnson/markdownlint) which this wraps, so I'd prefer to leave it as `markdownlint` to keep consistent with that. It just adds Electron-specific rules on top of `markdownlint`, which is why I'm leaving the name specific to Electron (it's not generic like the others).